youtube_dl/extractor/wdr.py

```diff
@@ -16,59 +17,45 @@
 
 
 class WDRIE(InfoExtractor):
-    _PLAYER_REGEX = '-(?:video|audio)player(?:_size-[LMS])?'
-    _VALID_URL = r'(?P<url>https?://www\d?\.(?:wdr\d?|funkhauseuropa)\.de/)(?P<id>.+?)(?P<player>%s)?\.html' % _PLAYER_REGEX
-
+    _PLAYER_REGEX = 'https?://deviceids-medstdp.wdr.de/ondemand/.+?/.+?\.js'
```

Comment:
It would be more reasonable to extract mp4 using the webpage from old approach (e.g. `http://www1.wdr.de/mediathek/video/sendungen/hier_und_heute/videostreetfoodpioniere100-videoplayer_size-L.html`) since it contains different mp4 qualities including one in this JSON. Moreover there is an m3u8 that should be extracted as well,